### PR TITLE
Refactor: removes models bucket

### PR DESCRIPTION
**Pull request by GuillaumeTech**
- [ ] I wrote tests for the feature
- [ ] I documented the feature if needed

If the feature is a refactor, please explain why your way is better

### CODE REVIEW

File: botfront/imports/lib/utils.js [hunk 1]
```diff
@@ -80,15 +80,6 @@ export const getProjectModelLocalFolder = () => process.env.MODELS_LOCAL_PATH ||
 
 export const getProjectModelLocalPath = projectId => path.join(getProjectModelLocalFolder(), getProjectModelFileName(projectId, 'tar.gz'));
 
-export function uploadFileToGcs(filePath, bucket) {
-    const { Storage } = require('@google-cloud/storage');
```
Comment (pheel) [inline botfront/imports/lib/utils.js:84]: Make sure to uninstall this.

File: botfront/imports/api/instances/instances.methods.js [hunk 2]
```diff
@@ -341,15 +346,18 @@ if (Meteor.isServer) {
...
-                            }
+                    }
+
+                    const trainingWebhook = await getAfterTrainingWebhook();
+                    if (trainingWebhook?.url && trainingWebhook?.method) {
+                        const { namespace } = await Projects.findOne({ _id: projectId }, { fields: { namespace: 1 }})
+                        const body = {
+                            projectId,
+                            namespace,
+                            model: Buffer.from(trainingResponse.data).toString('base64'),
```
Comment (pheel) [inline botfront/imports/api/instances/instances.methods.js:357]: I'm not sure encoding it to text is needed/good, but I guess that's what's already done in `deploy.model`... Which makes this think, this should go outside of `rasa.train`,  in the same file as `deploy.model`, and share the common parts.

File: botfront/private/defaults/private.dev.yaml [hunk 3]
```diff
@@ -52,3 +52,7 @@ webhooks:
     name: ReportCrash
     url: ''
     method: POST
+  afterTraining:
```
Comment (pheel) [inline botfront/private/defaults/private.dev.yaml:55]: "PostTraining" is more elegant, but doesn't really matter.

**Review by pheel — APPROVED** [review]

File: botfront/imports/lib/utils.js (hunk 1 above)
Comment (kenrilea) [reply botfront/imports/lib/utils.js:84]: moved to https://github.com/botfront/botfront/pull/856 so I don't have to re-reun the jobs on this branch and potentially delay the release

Comment (pheel) [reply botfront/imports/lib/utils.js:84]: Oops

Comment (kenrilea) [reply botfront/imports/lib/utils.js:84]: no problem, better to do it here really 🙂

File: botfront/imports/lib/utils.js [hunk 4]
```diff
@@ -259,6 +259,7 @@ if (Meteor.isServer) {
             if (!trainingWebhook.url || !trainingWebhook.method) {
                 return;
             }
+            console.log(trainingWebhook)
```
Comment (pheel) [inline botfront/imports/lib/utils.js:262]: :'(

**Pull request merged**